vendor/k8s.io/test-infra/pkg/flagutil/flagutil.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I don't think this is suppoed to be in this PR

Reply:
This was generated when I ran the update-vendor/validate-vendor, after merging the changes from the update-test-infra. I ran those commands as the validate-vendor checks were still failing on this pr and thats why I included expected.yaml, BUILD.bazel, flagutil.go and modules.txt. Should i drop just this file or all 4 of the files which were included?

Reply:
yeah, ideally you can soft reset your branch to the oldest commit in it ~1, create one commit and rebase it onto master.

One problem with merging changes from the basebranch in rather than rebasing is that they might show up in the PR which is confusing.

Reply:
Resolving - Added an extra module, therefore we need these files